### PR TITLE
PE-4392: fix(update drive table when renaming a file)

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/5mr34g0lsp700

### CODE REVIEW

File: lib/pages/drive_detail/components/drive_detail_data_list.dart [hunk 1]
```diff
@@ -147,8 +147,12 @@ Widget _buildDataListContent(
   bool isMultiselecting,
 ) {
   return LayoutBuilder(builder: (context, constraints) {
+    final driveDetailCubitState = context.read<DriveDetailCubit>().state;
+    final equtableBust = driveDetailCubitState is DriveDetailLoadSuccess
```
Comment (kunstmusik) [inline lib/pages/drive_detail/components/drive_detail_data_list.dart:151]: typo: equatable

File: lib/blocs/drive_detail/drive_detail_cubit.dart [hunk 2]
```diff
@@ -398,7 +398,9 @@ class DriveDetailCubit extends Cubit<DriveDetailState> {
     _refreshSelectedItem = true;
 
     if (state is DriveDetailLoadSuccess) {
-      emit((state as DriveDetailLoadSuccess).copyWith());
+      logger.i('refreshing drive data table');
```
Comment (kunstmusik) [inline lib/blocs/drive_detail/drive_detail_cubit.dart:401]: Probably can remove this logger call

File: lib/blocs/drive_detail/drive_detail_state.dart [hunk 3]
```diff
@@ -69,8 +72,10 @@ class DriveDetailLoadSuccess extends DriveDetailState {
     bool? hasFoldersSelected,
     ArDriveDataTableItem? selectedItem,
     List<ArDriveDataTableItem>? currentFolderContents,
+    Key? equatableBust,
   }) =>
       DriveDetailLoadSuccess(
+        forceRebuildKey: equatableBust ?? forceRebuildKey,
```
Comment (kunstmusik) [inline lib/blocs/drive_detail/drive_detail_state.dart:78]: modify to `forceRebuildKey: forceRebuildKey ?? this.forceRebuildKey`

Comment (thiagocarvalhodev) [reply lib/blocs/drive_detail/drive_detail_state.dart]: https://github.com/ardriveapp/ardrive-web/pull/1558/commits/bbedabff5ab49dd1390630f9e81968a6f0ebd453

File: lib/blocs/drive_detail/drive_detail_state.dart [hunk 4]
```diff
@@ -69,8 +72,10 @@ class DriveDetailLoadSuccess extends DriveDetailState {
     bool? hasFoldersSelected,
     ArDriveDataTableItem? selectedItem,
     List<ArDriveDataTableItem>? currentFolderContents,
+    Key? equatableBust,
```
[the remark this answers is not in the document]
Comment (thiagocarvalhodev) [reply lib/blocs/drive_detail/drive_detail_state.dart]: https://github.com/ardriveapp/ardrive-web/pull/1558/commits/bbedabff5ab49dd1390630f9e81968a6f0ebd453

File: lib/blocs/drive_detail/drive_detail_cubit.dart (hunk 2 above)
Comment (thiagocarvalhodev) [reply lib/blocs/drive_detail/drive_detail_cubit.dart]: https://github.com/ardriveapp/ardrive-web/pull/1558/commits/bbedabff5ab49dd1390630f9e81968a6f0ebd453

**Review by kunstmusik — DISMISSED** [review]

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**